kpatch-build/create-diff-object.c

```diff
@@ -69,6 +69,8 @@
 		printf(format, ##__VA_ARGS__); \
 })
 
+extern char *find_exported_symbol_objname(const char *fname, char *symname);
```

Comment:
Yeah I think that makes more sense, instead of having the out-of-place extern declaration here it could go in lookup.h. Technically the current parse-mod-symvers stuff can stay its own file, but we can move it to lookup.c, but how does licensing work in that case?